test/types/enum/operations/order/enumOrder.chpl

```diff
@@ -17,4 +17,4 @@ test(color.green, 1);
 test(color.blue, 2);
 
 // try an illegal case:
-writeln(chpl__orderToEnum(4, color));
+writeln(chpl__orderToEnum(four, color));
```

Comment:
Possible to have one of these tests try an illegal case against the low bound (if they don't already)?

Reply:
Commit https://github.com/chapel-lang/chapel/pull/17504/commits/ed214bc096bb958a19605a6c0269be1ce3e4508a changes enumOrder3.chpl to do this (since that part of each of the first three tests is essentially redundant with the others).